packages/core/src/global/mixins/_form-items.scss

```diff
@@ -150,26 +150,34 @@
 }
 
 @mixin icon {
-  width: 1rem;
-  height: 1rem;
-  position: absolute;
+  display: flex;
+  align-items: center;
+  justify-content: center;
   line-height: 1;
   text-align: center;
   fill: currentColor;
-  // optical correction, subtract 1px
-  top: $ray-field-v-spacing - rem(1px);
 
   path {
     fill: currentColor;
   }
 }
 @mixin icon-support($class) {
   &--with-icon-left {
+    &.#{$ray-class-prefix}#{$class}--active,
```

Comment:
Some adjustments for the label and input to appear correctly when the input is active or has value